roboteam_ai/src/ApplicationManager.cpp

```diff
@@ -82,6 +82,8 @@ void ApplicationManager::runOneLoopCycle() {
         strategy = factory.getTree(BTFactory::getCurrentTree());
         Status status = strategy->tick();
         this->notifyTreeStatus(status);
+
+        rtt::ai::coach::OffensiveCoach::calculateNewPositions();
```

Comment:
so you are always calculating positions, regardless of the strategy. This feels weird. What positions are you calculating?

Reply:
Just generally good offensive positions, good positions to receive the ball and shoot at the goal, and preferably a bit close to the opponents goal